pyseq.py

```diff
@@ -128,6 +139,24 @@ def __init__(self, item):
         self.head = self.name
         self.tail = ''
 
+    def __eq__(self, other):
```

Comment:
Yeah, I think that's what it was but I found it odd because the comparison should have been using str's __eq__ which I think should have found the two items equal based on the string in Item itself.